src/Voron/StorageEnvironment.cs

```diff
@@ -875,9 +875,53 @@ internal void TransactionCompleted(LowLevelTransaction tx)
             }
         }
 
+        public SizeReport GenerateSizeReport()
+        {
+            long journalsSize = 0;
+            foreach (var journal in Journal.Files.ToList())
```

Comment:
Why do we have the allocation on `ToList` here?  The `Files` is immutable.

Reply:
done